docs/envvars.rst

```diff
@@ -74,6 +74,13 @@ applicable.
       - xonsh.environ.FORMATTER_DICT  
       - Dictionary containing variables to be used when formatting PROMPT and TITLE 
         see `Customizing the Prompt <tutorial.html#customizing-the-prompt>`_.
+    * - HISTCONTROL
+      - ``''``
+      - A comma-separated list of options that determine if commands are saved
```

Comment:
This should probably be a set then, not a string. (I assume that it doesn't make sense for options to be repeated.) You'll most likely want to write your own Environment validators and to- and from-string conversion functions.